source/voice_control/thirteen-usd-voice-remote.markdown

```diff
@@ -86,6 +86,15 @@ Before you can use this device with Home Assistant, you need to install a bit of
    - While you are speaking, the blue LED is pulsing.
    - Once the intent has been processed, the LED lights up in green and Home Assistant confirms the action.
 
+## Enabling updates
```

Comment:
Thanks for the PR, however, as of the last release, this is no longer needed.

ESPHome projects are now able to ship pre-build updates to the devices direct, without the need for installing the add-on, and without the need for adoption or modifying configuration.

../Frenck